### ISSUE TITLE
16 bit flac output conversion?

### ISSUE THREAD

**Issue by Mixerrog** [faroit/stempeg#43]
Is there a way to convert the 4 stem output files from the new Open-Unmix UMX using Stempeg to output 16 bit flac files instead of the 24 bit flac files I am currently getting using it?

Thank, Rog

**Comment by faroit** [comment]
@Mixerrog yes, you are right. 24bit seems to be the default bit-depth for flac files. You would have to pass `sample_fmt=s16` to ffmpeg-python to enable this but stempeg has currently no way to pass this. Feel free to add a pull request!

**Comment by Mixerrog** [comment]
Don't see how to add a pull request but just want to be able to output 16 bit flac stems from the new Open-Unmix UMX version please instead of the default 24 bit stems that Stempeg is giving me?

Thanks, Rog

**Comment by faroit** [comment]
> Don't see how to add a pull request but just want to be able to output 16 bit flac stems from the new Open-Unmix UMX version please instead of the default 24 bit stems that Stempeg is giving me?

stempeg is open-source and you can just modify it to your liking (and add a PR when you like). You probably would want to add it to this line: https://github.com/faroit/stempeg/blob/master/stempeg/write.py#L636
